src/store/imageBuilderApi.ts

```diff
@@ -295,8 +295,8 @@ export type CustomRepository = {
 };
 export type OpenScap = {
   profile_id: string;
-  profile_name?: string;
-  profile_description?: string;
+  profile_name: string;
+  profile_description: string;
```

Comment:
This looks like a manual change to the generated code which means that when the API schema gets updated and the `npm run api` is run the change will get overwritten and lost.

If the change is needed it should be done directly in the image-builder API so it gets propagated consistently. Do the name and description need to be required?

Reply:
Is it a manual change? Travis should catch it then! But I think Travis is broken :sob:

Reply:
Great point! As we are planning to no longer depend on Travis anyway, we should probably add the check to our dev check actions. I'll make a note and take a look at it.

Reply:
Actually - general question about the name and description in the API - do we even use it in the code? What I mean is, I've tried to add an OSCAP profile to a mock to see what I get on Recreate.

Updated a compose fixture with:

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/f2926de6-b152-469b-b82b-070e59a470e5)

But what I see on Recreate is:

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/38a0a16c-5152-460d-9c1d-00ebd550fa87)

And now I'm a tad bit confused :sweat_smile:

Reply:
I run the npm run api and its adding those rows automatically, I didnt add it manually .

Reply:
Yup, the rows should be added automatically. I was wondering about the removed question marks in one of the previous commits. But that could have also been a mismatch between schema and API version before :+1:  :)

Reply:
Regarding the check that's done in travis, you can move it to to a github action easily, it's just a bash oneliner.

Reply:
Tadaa :tada: https://github.com/RedHatInsights/image-builder-frontend/pull/1510